### DISCUSSION TITLE
I encountered a FileNotFoundException error during the installation of ChatApp.

### DISCUSSION THREAD

**Discussion by derekzhou2015** [ScoopInstaller/Scoop#5480] [Help and Questions]
PS C:\Users\Derek> scoop install chatbox
Installing 'chatbox' (0.4.0) [64bit] from anderlli0053_DEV-tools bucket
Loading chatbox_0.4.0_x64_en-US.msi from cache
Checking hash of chatbox_0.4.0_x64_en-US.msi ... ok.
Extracting chatbox_0.4.0_x64_en-US.msi ... done.
Running installer script...
Import-Module : The specified module 'D:\Users\Derek\SCOOP\buckets\apps\scripts\AppsUtils.psm1' was not loaded because no valid module file was found in
any module directory.
At line:2 char:1
+ Import-Module $(Join-Path $(Find-BucketDirectory -Root -Name apps) sc ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

I can't find the app ,scripts .

**Comment by rashil2000** [comment]
Please report this issue in the relevant bucket repository, i.e. anderlli0053_DEV-tools
